metadetect/lsst/tests/test_lsst_conventions.py

```diff
@@ -8,12 +8,20 @@
 from metadetect.lsst.metadetect import get_fitter
 import metadetect.lsst.measure
 from metadetect.lsst import util
+from lsst.utils import getPackageDir
 
 logging.basicConfig(
     stream=sys.stdout,
     level=logging.INFO,
 )
 
+try:
+    getPackageDir('descwl_shear_sims')
+    getPackageDir('descwl_coadd')
```

Comment:
Better to just try to import these directly and catch the import error; that will work even if the packaging is not handled by EUPS someday.